.github/workflows/tx-pull.yml

```diff
@@ -46,33 +48,10 @@ jobs:
         # See https://git-scm.com/docs/git-add#_options
         add_options: '-A'
 
-        # Optional. Options used by `git-push`.
-        # See https://git-scm.com/docs/git-push#_options
-        push_options: '--force'
```

Comment:
no longer force pushing for sanity.

Reply:
I'm more okay with the removal of this one than the others.

Reply:
shouldn't we be checking for `dirty` always `fetch` before creating commit?